uobtheatre/productions/schema.py

```diff
@@ -223,6 +225,9 @@ def resolve_total_tickets_sold(self, info):
     def resolve_content_warnings(self, info):
         return self.warnings_pivot.order_by("warning__short_description").all()
 
+    def resolve_excerpt(self, info):
```

Comment:
Graphene should automatically use properties as resolvers so you should be able to remove this function